projects/Generic/filesystem/usr/lib/systemd/system/rtk_bt.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Realtek RTL8723BS bluetooth service
+ConditionPathExists=/dev/ttyS1
```

Comment:
`/dev/ttyS1` is going to exist on pretty much every PC, regardless of whether it has this chipset or not, and as such it isn't a suitable condition on which to base the loading of this driver.